bundles/core/org.eclipse.smarthome.core.id.test/META-INF/MANIFEST.MF

```diff
@@ -1,21 +1,16 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: Eclipse SmartHome ID Tests
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: org.eclipse.smarthome.core.id.test;singleton:=true
-Bundle-Vendor: Eclipse.org/SmartHome
-Bundle-Version: 0.10.0.qualifier
-Fragment-Host: org.eclipse.smarthome.core.id
-Import-Package: 
- groovy.lang,
- org.codehaus.groovy.reflection,
- org.codehaus.groovy.runtime,
- org.codehaus.groovy.runtime.callsite,
- org.codehaus.groovy.runtime.typehandling,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.test,
- org.eclipse.smarthome.test.storage,
- org.hamcrest;core=split,
- org.junit;version="4.0.0",
- org.osgi.framework
-Automatic-Module-Name: org.eclipse.smarthome.core.id.test
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome ID Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.eclipse.smarthome.core.id.test;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.10.0.qualifier
+Fragment-Host: org.eclipse.smarthome.core.id
+Import-Package: 
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.test,
+ org.eclipse.smarthome.test.storage,
+ org.hamcrest;core=split,
```

Comment:
Is using Hamcrest matchers still preferred or is its faith going to be the same as Groovy?
There's the trade off between readability and adding an extra dependency.

Reply:
Hm, why do we need them at all?
Can't we change `assertThat(uuid, is(equalTo("123")));` to `assertEquals("123", uuid)` (static import of Assert.* from JUnit)?
IMHO `assertEquals` is readable :wink: 
WDYT?

Reply:
Hamcrest may be overkill in a project like this with just a few assertions.

When projects grow and start to contain more and complexer tests, it's easier to immediately understand the intention of tests when you can read the Hamcrest assertThat statements like normal written sentences.

Though you can also use streams with predicates nowadays which make up for good alternatives for methods like anyOf/allOf in which Hamcrest really had an edge in the pre-Java-8 days.

Reply:
It seems in this test case there is no need to use Hamcrest at all.
Would you like to remove the Hamcrest usage for that test now or do you prefer to leave it in place?

Reply:
I'll remove it from this project since it's just a few asserts.